src/app/aether-application/application-edit/application-edit.component.ts

```diff
@@ -139,9 +139,6 @@ export class ApplicationEditComponent
             bs,
             route,
             router,
-            'Enterprises-2.0.0',
```

Comment:
I see you moved these out in to a separate file

Reply:
yes, that way we don't need to duplicate between the `list` and `edit` views. I'm trying to eradicate all the references to the "old" ones.